src/Pages/Events/EventCard.js

```diff
@@ -1,53 +1,65 @@
 import React from 'react';
-import { Container, Row, Col } from 'reactstrap';
+import { Row } from 'reactstrap';
 import { getDateWithSlashes } from '../../APIFunctions/Event';
-import { mapPinSymbol, clockSymbol} from '../Overview/SVG';
+import { mapPinSymbol, clockSymbol } from '../Overview/SVG';
+// import { sourcerepo } from 'googleapis/build/src/apis/sourcerepo';
 
 function EventCard(props) {
   const {
     title,
-    description,
     eventDate,
     eventLocation,
     startTime,
     endTime,
     handleClick,
-    imageURL
+    imageURL,
+    noImageURL = 'https://www.freeiconspng.com/uploads/no-image-icon-11.PNG',
```

Comment:
it would be better to send `imageURL` as a prop that already handles if it isnt defined or not in the parent component
```suggestion
```